tasks/publish_statistics.rb

```diff
@@ -7,11 +7,18 @@
   performance_gateway = PerformancePlatform::Gateway::PerformanceReport.new
   volumetrics_gateway = PerformancePlatform::Gateway::Volumetrics.new(date: args[:date])
   volumetrics_presenter = PerformancePlatform::Presenter::Volumetrics.new(date: args[:date])
+  completion_rate_gateway = PerformancePlatform::Gateway::CompletionRate.new(date: args[:date], period: "day")
+  completion_rate_presenter = PerformancePlatform::Presenter::CompletionRate.new(date: args[:date])
```

Comment:
shameless plug to my way of [not duplicating these tasks](https://github.com/alphagov/govwifi-logging-api/blob/master/tasks/publish_statistics.rb#L13). Otherwise I'm worried it might get hard to read quite soon.